packages/components/src/components/Spacer/Spacer.tsx

```diff
@@ -43,6 +43,15 @@ export type SpacerProps = {
   horizontal?: boolean
 }
 
+/**
+ * Convenience function that accepts a spacing size abbreviation and returns the corresponding
+ * spacing token
+ */
+export function getSpacingToken(size: SpacerSize): number {
```

Comment:
Don't feel strongly that anything be changed, but raising the question of if this should be in `utils` somewhere instead of the component.

Reply:
Created a new `utils/tokens.ts` file and added to the utils index export. Moved this function there.